log/log.go

```diff
@@ -93,30 +92,36 @@ func InitSpaceMeshLoggingSystem(dataFolderPath string, logFileName string) {
 
 // public wrappers abstracting away logging lib impl
 
-// Standard info level logging
+// Info prints formatted info level log message.
 func Info(format string, args ...interface{}) {
 	ulogger.Logger.Info(format, args...)
 }
 
-// Standard debug level logging
+// Debug prints formatted debug level log message.
 func Debug(format string, args ...interface{}) {
 	ulogger.Logger.Debug(format, args...)
 }
 
-// Standard error level logging
+// Error prints formatted error level log message.
 func Error(format string, args ...interface{}) {
 	ulogger.Logger.Error(format, args...)
 }
 
-// Standard warning level logging
+// Warning prints formatted warning level log message.
 func Warning(format string, args ...interface{}) {
 	ulogger.Logger.Warning(format, args...)
 }
 
-func PrettyId(id string) string {
+// Fatal prints formatted critical level log message and calls os.Exit(1).
+func Fatal(format string, args ...interface{}) {
```

Comment:
@kare I'm not sure this is needed here. In case of an Error that should cause the app to terminate, we'd like to exit it in a graceful way so services can be shut down, log files flushed, etc... We have an ExitApp bool channel this is designed for this. The other comments changes are good. thanks!